block-storage-attacher/tests/e2e/conf/developer.conf

```diff
@@ -50,11 +50,13 @@ HELM_CHART_REPO=$HELM_CHART_REPO
 HELM_CHART=$GOPATH/src/github.com/IBM/ibmcloud-storage-utilities/block-storage-attacher/helm/ibm-block-storage-attacher/
 
 PORTWORX_HELM_CHART=$GOPATH/portworx/helm/charts/portworx
-ETCD_SET1=etcd:https://portal-ssl495-47.bmix-dal-yp-b15b4c9e-cc62-4a8b-900d-895f414f2528.1152061296.composedb.com:18308
```

Comment:
Please change the `IMAGE_REGISTRY` in this file.

Reply:
This uses staging registry I Should change to staging